src/orion/core/worker/trial.py

```diff
@@ -516,5 +522,5 @@ def __exit__(self, exc_type, exc_value, traceback):
                 self._cm_experiment.release(self._cm_trial, "broken")
             elif self._cm_trial.status == "reserved":
                 self._cm_experiment.release(self._cm_trial)
-        except RuntimeError as e:
+        except AlreadyReleased as e:
```

Comment:
`client.release` should be adjusted accordingly: https://github.com/Epistimio/orion/blob/develop/src/orion/client/experiment.py#L477

Reply:
isnt it already adjusted ?


```
    def release(self, trial, status="interrupted"):
        """Release a trial.

        Release the reservation and stop the heartbeat.

        Experiment must be in writable ('w') or executable ('x') mode.

        Parameters
        ----------
        trial: `orion.core.worker.trial.Trial`
            Trial to reserve.
        status: str, optional
            Set the trial to given status while releasing the reservation.
            Defaults to 'interrupted'.

        Raises
        ------
        `RuntimeError`
            If reservation of the trial has been lost prior to releasing it.
        `AlreadyReleased`
            If reservation of trial was already released
        `ValueError`
            If the trial does not exist in storage.
        `orion.core.utils.exceptions.UnsupportedOperation`
            If the experiment was not loaded in writable mode.

        """
        ```

Reply:
Indeed!